### PR TITLE
[U] Migrate marked + vue compiler to prebuilt MDX

### PR DESCRIPTION
**Pull request by Mak1o**
Hardcoded markdown contents in `src/logic/config.ts` are also moved to `.md` files under `src/assets`, which are compiled to HTML at build time.

### CODE REVIEW

File: src/logic/config.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import urljoin from "url-join";
 
 export const backendHost = 'https://backend.one-among.us'
 // export const backendHost = 'http://localhost:43482'
-export const dataHost = 'https://data.one-among.us'
-// export const dataHost = 'http://localhost:8089'
+export const dataHost = 'https://mak1o.github.io/our-data/'
```
Comment (Mak1o) [inline src/logic/config.ts:5]: This should be changed back before merging.

Comment (Mak1o) [inline src/logic/config.ts:5]: This is for test and must be changed back before merging.

**Comment by Mak1o** [comment]
After compiling JSON5 to JSON in data repo, we no longer need JSON5 parser in frontend. But we still need JSON5 serializer in order to submit PRs.

Maybe we can move JSON5 serializer to server-side to remove JSON5 dependency in client-side.

**Review by hykilpikonna — APPROVED** [review]

**Comment by hykilpikonna** [comment]
> After [compiling JSON5 to JSON](https://github.com/hykilpikonna/our-data/pull/27/files#diff-858990bbf13afa1eb59bdc3483ce72befd12d3edd714865d84291b6f62553917R28) in data repo, we no longer need JSON5 parser in frontend. But we still need JSON5 serializer in order to submit PRs.
> 
> Maybe we can move JSON5 serializer to server-side to remove JSON5 dependency in client-side.

谢谢！我看看能不能把 JSON5 挪到 Java 后端那边

**Comment by hykilpikonna** [comment]
先合并啦，等 our-data 那边也部署上再把 dataHost 换回去好了

**Pull request merged**